github/pulls_reviews.go

```diff
@@ -35,9 +35,12 @@ func (p PullRequestReview) String() string {
 
 // DraftReviewComment represents a comment part of the review.
 type DraftReviewComment struct {
-	Path     *string `json:"path,omitempty"`
-	Position *int    `json:"position,omitempty"`
-	Body     *string `json:"body,omitempty"`
+	Path        *string `json:"path,omitempty"`
+	Position    *int    `json:"position,omitempty"`
+	Body        *string `json:"body,omitempty"`
+	CommitID    *string `json:"commit_id,omitempty"`
+	InReplyTo   *int64  `json:"in_reply_to,omitempty"`
```

Comment:
On this page: https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#list-comments-for-a-pull-request-review--code-samples
I see `in_reply_to_id` but I'm not finding `id_reply_to` anywhere on the page.

Where are you finding that field?

Also, I'm not finding `subject_type` anywhere on that page. Where is that field coming from?

Reply:
Oh, I see them now on this page :https://docs.github.com/en/rest/pulls/comments?apiVersion=2022-11-28#create-a-review-comment-for-a-pull-request

Sorry for the confusion.